birdhouse/read-configs.include.sh

```diff
@@ -81,28 +81,50 @@ read_env_local() {
 
 }
 
+source_conf_files() {
+  dirs=$1
+  conf_locations=$2
+  for adir in ${dirs}; do
+      if echo "$ALL_CONF_DIRS" | grep -qE "^\s*$adir\s*$"; then
+          continue
```

Comment:
Ah, handle/ignore repeated entries.  Could you add a comment for the next person that did not review this PR.